### PR TITLE
support split apks

### PR DESCRIPTION
**Pull request by CaiMiao**
what split apk you can download might depend on your gsf id (at least abi config apks)

### CODE REVIEW

File: playstore/playstore.py [hunk 1]
```diff
@@ -548,12 +548,19 @@ def download(self, package_name: str, file_name: str = None, download_obb: bool
...
             # The app already belongs to the account.
             temp_url = response.payload.deliveryResponse.appDeliveryData.downloadUrl
             cookie = response.payload.deliveryResponse.appDeliveryData.downloadAuthCookie[0]
             additional_files = [additional_file for additional_file in
                                 response.payload.deliveryResponse.appDeliveryData.additionalFile]
+            split_apks = [split_apk for split_apk in
+                          response.payload.deliveryResponse.appDeliveryData.split] if\
+                          response.payload.deliveryResponse.appDeliveryData.split else None
+            self.logger.error(type(split_apks))
+            #self.sys.exit(1)
```
Comment (CaiMiao) [inline playstore/playstore.py]: remove this

File: playstore/playstore.py [hunk 2]
```diff
@@ -619,6 +628,39 @@ def download(self, package_name: str, file_name: str = None, download_obb: bool
...
+            for cfg_apk in split_apks:
+
+                # Execute another query to get the actual file.
+                response = requests.get(cfg_apk.downloadUrl, headers=headers, cookies=cookies, verify=True, stream=True)
+
+                chunk_size = 1024
+                file_size = int(response.headers['content-length'])
+
+                cfg_apk_file_name = os.path.join(os.path.dirname(file_name),
+                                             '{0}.{1}.{2}.apk'.format(package_name, version_code, cfg_apk.name))
```
Comment (ClaudiuGeorgiu) [inline playstore/playstore.py]: Using the same naming convention as for `obb` files:
```suggestion
                                             '{0}.{1}.{2}.apk'.format(cfg_apk.name, version_code, package_name))
```

File: playstore/playstore.py [hunk 3]
```diff
@@ -548,12 +548,19 @@ def download(self, package_name: str, file_name: str = None, download_obb: bool
...
         if response.payload.deliveryResponse.appDeliveryData.downloadUrl:
             # The app already belongs to the account.
             temp_url = response.payload.deliveryResponse.appDeliveryData.downloadUrl
             cookie = response.payload.deliveryResponse.appDeliveryData.downloadAuthCookie[0]
             additional_files = [additional_file for additional_file in
                                 response.payload.deliveryResponse.appDeliveryData.additionalFile]
+            split_apks = [split_apk for split_apk in
+                          response.payload.deliveryResponse.appDeliveryData.split] if\
+                          response.payload.deliveryResponse.appDeliveryData.split else None
+            self.logger.error(type(split_apks))
```
Comment (ClaudiuGeorgiu) [inline playstore/playstore.py]:
```suggestion
```

File: playstore/playstore.py (hunk 1 above)
Comment (ClaudiuGeorgiu) [inline playstore/playstore.py]:
```suggestion
```

**Pull request merged**